public/components/event_analytics/explorer/events_views/data_grid.tsx

```diff
@@ -262,7 +283,7 @@ export function DataGrid(props: DataGridProps) {
           toolbarVisibility={{
             showColumnSelector: {
               allowHide: false,
-              allowReorder: true,
+              allowReorder: explorerFields.selectedFields.length > 0,
```

Comment:
If by default only have timestamp and _source for initial state, then there's no selected fields explorerFields.selectedFields, but datagrid should still allow user to reorder timnestamp and _source is it?

Reply:
Here i'm actually disabling reordering when no fields are selected, so that no reordering is done when the default columns appear in data grid. The reason I do this is because the default columns are 'artificial fields' and I don't want them to appear as objects in the sidebar.

Reply:
then it doesn't match with existing behavior in discover

Reply:
existing behavior is discover is bugged a little bit, the default columns don't get reordered when done through the data grid even though the names move and the sidebar also has some inconsistencies. I extrapolated that since the default columns don't actually move themselves, then I should disable the ability to attempt at doing so.

Reply:
existing behavior is discover is bugged a little bit, the default columns don't get reordered when done through the data grid even though the names move and the sidebar also has some inconsistencies. I extrapolated that since the default columns don't actually move themselves, then I should disable the ability to attempt at doing so.